package.json

```diff
@@ -1,46 +1,39 @@
 {
+  "name": "visual-editing-monorepo",
   "private": true,
   "scripts": {
     "build": "turbo run build --filter='./packages/*'",
     "build:astro": "turbo run build --filter=./apps/astro",
-    "dev": "turbo run dev --filter=./apps/* --filter=!./apps/next-server-only --filter=!./apps/next-no-cache",
-    "dev:astro": "turbo run dev --filter=./apps/astro --filter=./apps/studio",
-    "dev:next": "turbo run dev --filter=./apps/next --filter=./apps/studio",
-    "dev:nuxt": "turbo run dev --filter=./apps/nuxt --filter=./apps/studio",
-    "dev:page-builder-demo": "turbo run dev --filter=./apps/page-builder-demo --filter=./apps/studio",
-    "dev:svelte": "turbo run dev --filter=./apps/svelte --filter=./apps/studio",
+    "dev": "turbo run build --filter='./packages/*' && turbo watch dev --filter='./packages/*' --filter=./apps/* --filter=!next-no-cache --filter=!next-server-only --filter=!next-with-i18n",
```

Comment:
Chaining `turbo run build` with `turbo watch dev` commands are only necessary until `@sanity/pkg-utils` can handle `.d.ts` generation in its `watch` command. Once that is supported we can go back to just `turbo run dev` 🙌

Reply:
That's great. For now the slightly slower startup time is definitely worth the HMR improvements.